ianshulx/React-projects-for-beginners
# Tip-Calculator App

**Title:** Add Vite Tip Calculator App with Tailwind CSS

**Description:** This PR introduces a responsive Tip Calculator app built with Vite and styled using Tailwind CSS. The app allows users to quickly calculate tips based on bill amounts and desired tip percentages.

**Key Features:**
User-friendly Input: Allows users to enter a bill amount and select a tip percentage.
Real-time Calculation: Updates tip amount and total bill in real-time as values are adjusted.
Responsive Design: Uses Tailwind CSS for a mobile-friendly and visually appealing layout.

**How to Test:**
Clone the repository and navigate to the project folder.
Run **npm install** to install dependencies.
Start the development server with **npm run dev**.

Test the calculator by entering different bill amounts and tip percentages to verify accuracy.
